packages/example-studio/components/ProteinInput/ProteinInput.js

```diff
@@ -0,0 +1,263 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import Fieldset from 'part:@sanity/components/fieldsets/default'
+import PatchEvent, {set} from '@sanity/form-builder/PatchEvent'
+import {Viewer, io, viewpoint, color} from 'bio-pv'
+import {debounce, values, get} from 'lodash'
+import Select from 'part:@sanity/components/selects/default'
+import TextField from 'part:@sanity/components/textfields/default'
+import Button from 'part:@sanity/components/buttons/default'
+import Spinner from 'part:@sanity/components/loading/spinner'
+
+// import fetch from 'fetch'
+
+const pdbs = [
+  {
+    title: ''
+  },
+  {
+    title: 'Adenovirus',
+    pdb: '3IYN'
+  },
+  {
+    title: 'Insect Flight Muscle',
+    pdb: '2W49'
+  },
+  {
+    title: '4HHB',
+    pdb: '4HHB'
+  },
+  {
+    title: 'Crystal structure of Myosin VIIa ',
+    pdb: '5WSU'
+  },
+  {
+    title: 'Protonation-mediated structural flexibility in the F conjugation regulatory protein.',
+    pdb: '2G7O'
+  },
+  {
+    title: 'Escherichia coli Hfq-RNA',
+    pdb: '4PNO'
+  }
+]
+
+const options = {
+  width: 'auto',
+  height: '500',
+  antialias: true,
+  fog: true,
+  outline: true,
+  quality: 'high',
+  style: 'phong',
+  selectionColor: 'white',
+  transparency: 'screendoor',
+  background: '#fff',
+  animateTime: 500,
+  doubleClick: null
+}
+
+export default class ProteinInput extends React.Component {
+  static propTypes = {
+    type: PropTypes.shape({
+      title: PropTypes.string
+    }).isRequired,
+    value: PropTypes.object,
+    level: PropTypes.number,
+    onChange: PropTypes.func.isRequired
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: (props.value && props.value.pdb),
+      pdb: (props.value && props.value.pdb) || '1R6A',
+      newPdb: false
+    }
+  }
+
+  componentDidMount() {
+    this.structure = ''
+    this.fetchPdb(this.state.pdb)
+    // this.loadPDB('1r6a')
+    this.viewer = new Viewer(this._viewerElement, options)
```

Comment:
From the source code it looks like its `viewer.destroy()`
https://github.com/biasmv/pv/blob/master/src/viewer.js#L1167